app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'post_id'
+  has_many :likes, foreign_key: 'post_id'
+
+  def update_post_counter
+    user.update(posts_counter: user.posts.count)
+  end
+
+  def recent_posts
+    Post.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] Kindly consider the post model has a method that returns the 5 most recent comments for a given post.

Reply:
Fixed.